src/content/structured/components/badge/code.mdx

```diff
@@ -0,0 +1,755 @@
+      <IcBadge text-label="1" slot="badge" variant="info" />
+      Number
+    </IcButton>
+  </div>
+  <div style={{padding:"16px"}}>
+    <IcButton variant="secondary" style={{marginRight:"16px"}}>
+      <IcBadge type="dot" slot="badge" variant="info" />
+      Dot
+    </IcButton>
+  </div>
+  <div style={{padding:"16px"}}>
+    <IcButton variant="secondary" style={{marginRight:"16px"}}>
+      <IcBadge type="icon" slot="badge" variant="info">
+        <SlottedSVG 
+          width="24px"
+          height="24px"
+          slot="badge-icon" 
+          viewBox="0 0 24 24"
+        >
+          <path d="M0 0h24v24H0V0z" fill="none" />
+          <path
+            d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z"
+          />
+        </SlottedSVG>
+      </IcBadge>
+      Icon
+    </IcButton>
+  </div>
+</ComponentPreview>
+
+### Max number
+
+
+export const snippetsMaxNumber = [
+  {
+    language: "Web component",
+    snippet: `<div style="padding:16px;">
+  <ic-button variant="secondary" style="margin-right:16px">
+    <ic-badge
+      text-label="1000"
+      slot="badge"
+      size="small"
+      max-number="9"
+      variant="info"
+    >
+    </ic-badge>
+    Small
+  </ic-button>
+  <ic-button variant="secondary" style="margin-right:16px">
+    <ic-badge 
+      text-label="1000" 
+      max-number="9" 
+      slot="badge" 
+      variant="info"
+    >
+    </ic-badge>
+    Default
+  </ic-button>
+  <ic-button variant="secondary">
+    <ic-badge
+      text-label="1000"
+      slot="badge"
+      size="large"
+      max-number="9"
+      variant="info"
+    >
+    </ic-badge>
+    Large
+  </ic-button>
+</div>`,
+  },
+  {
+    language: "React",
+    snippet: `<div style={{padding:"16px"}}>
+  <IcButton variant="secondary" style={{marginRight:"16px"}}>
+    <IcBadge
+      text-label="1000"
+      slot="badge"
+      size="small"
+      max-number="9"
+      variant="info"
+    >
+    </IcBadge>
+    Small
+  </IcButton>
+  <IcButton variant="secondary" style={{marginRight:"16px"}}>
+    <IcBadge
+      text-label="1000"
+      max-number="9"
+      slot="badge"
+      variant="info"
+    >
+    </IcBadge>
+    Default
+  </IcButton>
+  <IcButton variant="secondary"
+    ><IcBadge
+      text-label="1000"
+      slot="badge"
+      size="large"
+      max-number="9"
+      variant="info"
+    >
+    </IcBadge>
+    Large
+  </IcButton>
+</div>`,
+  },
+];
+
+<ComponentPreview snippets={snippetsMaxNumber}>
+  <div style={{padding:"16px"}}>
+    <IcButton variant="secondary" style={{marginRight:"16px"}}>
+      <IcBadge
+        text-label="1000"
+        slot="badge"
+        size="small"
+        max-number="9"
+        variant="info"
+       />
+      Small
+    </IcButton>
+    <IcButton variant="secondary" style={{marginRight:"16px"}}>
```

Comment:
I think we probably just need the default example for max number as there's already a separate example for sizes